pythainlp/tools/__init__.py

```diff
@@ -22,7 +22,9 @@ def get_pythainlp_data_path() -> str:
     """
     Return full path where PyThaiNLP keeps its (downloaded) data
     """
-    path = os.path.join(os.path.expanduser("~"), PYTHAINLP_DATA_DIR)
+    path = os.environ('PYTHAINLP_DATA_DIR',
+                      os.path.join("~", PYTHAINLP_DATA_DIR))
```

Comment:
Error `TypeError: '_Environ' object is not callable`

https://travis-ci.org/PyThaiNLP/pythainlp/builds/562963089#L421

Reply:
latest commit should fix the problem